### PR TITLE
Added layout of app-card

### PR DESCRIPTION
**Pull request by teifler**

### CODE REVIEW

File: styles/components/app-card.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<section class="app-card">
+  <button class="app-card__bookmark"></button>
+  <h4 class="app-card__title">Question</h4>
```
Comment (ttaannjjaa) [inline styles/components/app-card.html]: because of semantic html i think here is h2 needed 😃

File: styles/components/app-card.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Debitis beatae velit nesciunt, quod sint aut libero
+      architecto aliquid vel tempora.
+    </details>
+    <ul class="app-card__list">
+      <li class="app-card__list__item"><button class="app-card__button">Tag 1</button></li>
+      <li class="app-card__list__item"><button class="app-card__button">Tag 2</button></li>
+      <li class="app-card__list__item"><button class="app-card__button">Tag 3</button></li>
+      <li class="app-card__list__item"><button class="app-card__button">Tag 4</button></li>
+    </ul>
+  </div>
```
Comment (ttaannjjaa) [inline styles/components/app-card.html]: randomly i think a closing div lost

File: styles/components/app-card.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.app-card {
+  background-color: white;
+  padding: 10px;
+  position: relative;
+  border: 1px solid black;
+  width: 40%;
```
Comment (ttaannjjaa) [inline styles/components/app-card.css]: i think, this might be too less with 40%

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Great job 👍😊

File: styles/components/app-card.html (hunk 1 above)
Comment (teifler) [reply styles/components/app-card.html]: Thanks for checking! I fixed the issue :)

**Pull request merged**